src/main/java/com/cs56fitnessapp/models/MealTitle.java

```diff
@@ -0,0 +1,20 @@
+package com.cs56fitnessapp.models;
+
+/**
+ * @Author jimweezy
+ * Created: 10/24/17
+ * Last changes: 10/24/17 at 5:37 AM
+ **/
+public enum MealTitle {
+    QUICK_MEAL("Quick Meal"), BREAKFAST("Breakfast"), LUNCH("Backstroke"), DINNER("Dinner"), DESSERT("Dessert");
```

Comment:
LUNCH("Backstroke") 😁

Reply:
What is the difference between quick meal and snack? 
If we're allow for the user quick input the property itself - will probably not below here. Let's stick with Snack only, then refactor, if needed.